src/main/java/it/pagopa/ecommerce/commons/domain/v1/pojos/BaseTransactionWithUserReceiptError.java

```diff
@@ -0,0 +1,51 @@
+package it.pagopa.ecommerce.commons.domain.v1.pojos;
+
+import it.pagopa.ecommerce.commons.documents.v1.TransactionUserReceiptData;
+import lombok.AccessLevel;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import lombok.experimental.FieldDefaults;
+
+/**
+ * <p>
+ * POJO for a transaction with an error adding user receipt
+ * </p>
+ *
+ * @see BaseTransactionWithRequestedUserReceipt
+ * @see TransactionUserReceiptData
+ */
+@ToString
+@EqualsAndHashCode(callSuper = true)
+@FieldDefaults(makeFinal = true, level = AccessLevel.PRIVATE)
+@Getter
+/*
+ * @formatter:off
+ *
+ * Warning java:S110 - This class has x parents which is greater than 5 authorized
+ * Suppressed because the Transaction hierarchy modeled here force BaseTransactionWithUserReceipt
+ * to be instantiated only starting from a TransactionClosed. The hierarchy dept is strictly correlated
+ * to the depth of the graph representing the finite state machine so can be accepted that hierarchy level
+ * is deeper than the max authorized level
+ *
+ * @formatter:on
+ */
+@SuppressWarnings("java:S110")
+public abstract class BaseTransactionWithUserReceiptError extends BaseTransactionWithRequestedUserReceipt {
```

Comment:
Given that this POJO doesn't add any data is it useful to keep it?

Reply:
Yes, I agree with you, fixed with https://github.com/pagopa/pagopa-ecommerce-commons/pull/62/commits/905e49352ef779f3f721ec17e7df0b76c12eeb84